ethabi/src/function.rs

```diff
@@ -49,6 +49,11 @@ impl Function {
 	pub fn decode_output(&self, data: &[u8]) -> Result<Vec<Token>> {
 		decode(&self.output_param_types(), &data)
 	}
+
+  /// Parses the ABI function input to a list of tokens.
+  pub fn decode_input(&self, data: &[u8]) -> Result<Vec<Token>> {
```

Comment:
The `ethabi` project uses tabbed indentations; can we update the formatting here to use tabs and ensure it is indented at the same level as the other functions?